src/error.rs

```diff
@@ -169,6 +169,26 @@ where T: Into<Kind> {
     InternalFrom(err, None).into()
 }
 
+#[macro_export]
```

Comment:
The `#[macro_export]` was the original problem here, this actually exports the macro out of the crate. Does this not work without this attribute?